### PR TITLE
Renames GAS to GAGS

### PR DESCRIPTION
**Pull request by ninjanomnom**
Greyscale Autogenerated Sprites -> GAS
Greyscale AutoGenerated Sprites -> GAGS

It was funny for a bit but it turns out when you name a system a noun that is in common usage in the same context, things get confusing.

### CODE REVIEW

File: code/datums/greyscale/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Greyscale Autogenerated Sprites (GAS)
+# Greyscale AutoGenerated Sprites (GAGS)
```
Comment (Fikou) [inline code/datums/greyscale/README.md:1]:
```suggestion
# Greyscale Auto-Generated Sprites (GAGS)
```

**Comment by tralezab** [comment]
Nooo! I cannot GAS my CBT every time I use your feature!!!!!

**Comment by tralezab** [comment]
[1]. Greyscale Autogenerated Sprites
[2]. Common Build Tool

**Comment by maxymax13** [comment]
how good are your greyscale auto generated reflexes

**Comment by interestingusernam3** [comment]
no ERP

**Review by spookydonut — APPROVED** [review]

**Pull request merged**